appservice/src/connectToGitHub.ts

```diff
@@ -110,9 +151,10 @@ async function getJsonRequest(url: string, requestOptions: WebResource, node: IA
  * @param label Property of JSON that will be used as the QuickPicks label
  * @param description Optional property of JSON that will be used as QuickPicks description
  * @param data Optional property of JSON that will be used as QuickPicks data saved as a NameValue pair
+ * @param quickPicks Optional property of QuickPickItems array that will be added to rather than returning a new one (side-effect)
  */
-function createQuickPickFromJsons(jsons: Object[], label: string, description?: string, data?: string[]): IAzureQuickPickItem<{}>[] {
-    const quickPicks: IAzureQuickPickItem<{}>[] = [];
+function createQuickPickFromJsons(jsons: Object[], label: string, description?: string, data?: string[], quickPicks?: IAzureQuickPickItem<{}>[]): IAzureQuickPickItem<{}>[] {
```

Comment:
nit: It feels odd to pass quickPicks here. Can you just append the result of this function to quickPicks outside of this function instead?